text/0077-issue-platform.md

```diff
@@ -0,0 +1,85 @@
+- Start Date: YYYY-MM-DD
+- RFC Type: informational
+- RFC PR: <link>
+- RFC Status: draft
+
+# Summary
+
+This rfc is intended to provide a high level via of the issue platform, it's 
+capabilities, how to interact with it and plans for the future.
+
+# Motivation
+
+The primary purpose of the issue platform is to allow different teams at Sentry to 
+detect problems relevant to our users in their data, and create new issues types from 
+that data.
+
+# High Level Concepts
+
+ - Event: A general term for data sent from SDKs to Sentry. This includes errors, transactions, profiles, replays, etc.
+ - Issue: An aggregation of events representing a unique problem we detected in the user’s system.
+ - Detector: A logical unit that analyzes data looking for problematic patterns and outputs found problems.
+ - Issue Occurrence: An event specifically associated with an issue. These are raised from detectors and sent to Sentry via the occurrences api.
+ - Fingerprint: An identifier of a unique occurrence. It’s used for grouping occurrences into issues.
+ - Issue Category: The broad category that an issue belongs to. `Error`, `Performance` and `Profile` are examples of this.
+ - Issue Type: The specific type of an issue. `N + 1 DB query` is an example of this. Type implies category.
+
+
+# Ingestion Schema
+
+Ingestion Schema
+Ingestion occurs via Kafka. The message format looks like
+
+ - id: str - A uuid representing the issue occurrence id.
+ - project_id: int.
+ - fingerprint: Sequence[str] - The fingerprint to uniquely identify a specific issue
+ - issue_title: str - The title of the occurrence. Will be used as issue title.
+ - subtitle: str - Used as the sub header on the issue.
+ - resource_id: str | None - An optional id that we can use to link to an external system
+ - evidence_data: Mapping[str, Any] - An arbitrary json blob of data for use with the frontend to render any custom components necessary for this issue type.
+ - evidence_display: A list of data to display in the ui and notifications. This will typically be displayed in a tabular format. Each entry is a dict in format
```

Comment:
```suggestion
 - evidence_display: A list of data to display in the ui and notifications. This will typically be displayed in a tabular format. Each entry is a dict using the format:
```